src/Controller/Api/GroupController.php

```diff
@@ -111,7 +111,7 @@ public function getGroupsAction(ParamFetcherInterface $paramFetcher)
      *
      * @View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=true)
      *
-     * @param $id
+     * @param string $id
```

Comment:
```suggestion
     * @param string|int $id Group identifier
```
I think we should we should add `int`, otherwise someone can add `string` type hint. Also Can we repeat desciption here?

Reply:
>I think we should we should add int, otherwise someone can add string type hint.

Parameter types can be certainly almost anything you want given the way the coercion works from the type declarations.

```
curl http://app/test/42/42/42/42
```
```php
/**
 * @Route("/test/{idUntyped}/{idString}/{idInteger}/{idFloat}")
 */
public function getTestAction($untyped, string $idString, int $idInteger, float $idFloat)
{
    var_dump($idUntyped); // string(2) "42"
    var_dump($idString); // string(2) "42"
    var_dump($idInteger); // int(42)
    var_dump($idFloat); // float(42)
}
```

Given that, I think we SHOULD declare these parameters as `string` in the next major.

>Also Can we repeat desciption here?

I'd prefer to let this task for a separate PR.